### DISCUSSION TITLE
Customize slug format

### DISCUSSION THREAD

**Discussion by m-emelchenkov** [statamic/cms#8261] [Q&A] [answered]
Hello! I'd like to adopt Statamic to one existing web project. The project has slugs with UTF-8 characters, space characters and underscores (due to non-English language). I dig into Statamic sources and not really sure how to adopt it. Statamic converts everything into lower-case latin with minus sign. Will it be easy to adopt or not, and where to look for this?

P.S.: the project's files has spaces, different capitalization and UTF-8 characters too.

**Comment by jasonvarga** [comment, marked as answer]
There's no way to customize the slug behavior at the moment.

We use Laravel's `Str::slug()` which ends up converting UTF-8 characters to ascii, replacing spaces with the separator, and other things.

**Comment by efc** [comment]
I'm sad about this, because there are cases where I would like to be able to have case-sensitive slugs with upper case letter, and `Str::slug()` nixes that as well.
